ksql-engine/src/main/java/io/confluent/ksql/services/KafkaTopicClientImpl.java

```diff
@@ -87,10 +88,6 @@ public void createTopic(
       ExecutorUtil.executeWithRetries(
           () -> adminClient.createTopics(Collections.singleton(newTopic)).all().get(),
           ExecutorUtil.RetryBehaviour.ON_RETRYABLE);
-    } catch (final InterruptedException e) {
```

Comment:
why?

Reply:
Usually InterruptedExceptions are special cased so that you can make sure they are being handled correctly, which unfortunately only a few Java libraries actually do. Essentially the idioms that I've seen recommended either:
1. re-throw the InterruptedException or
2. reset the interrupt flag on the thread and handle it

I think this actually falls in the second category, and we should be resetting the interrupt flag on the thread.

https://www.yegor256.com/2015/10/20/interrupted-exception.html has a good blog post about this

Reply:
Only really an issue if you're using interrupts for shutting down threads, which is generally not a good way to shut things down, (given issues with code swallowing interrupts!).

However, I've 'fixed' this code...